src/pages/Admin/Details/Form/DetailsForm.tsx

```diff
@@ -285,22 +280,20 @@ const DetailsForm: React.FC<DetailsFormProps> = ({ venue, worldId }) => {
     (venue) => venue.id === venue.worldId
   );
 
-  const backButtonOptionList =
-    filteredWorlds.reduce(
-      (
-        obj: Record<string, WithId<AnyVenue>> | null,
-        world: WithId<AnyVenue>
-      ) => {
-        if (venueId === world.id) {
-          return obj;
-        }
-
-        return { [world.id]: world, ...obj };
-      },
-      {}
-    ) ?? ALWAYS_EMPTY_OBJECT;
-
-  const parentSpace = filteredWorlds.find(({ id }) => id === venue?.parentId);
+  const backButtonOptionList = useMemo(
+    () =>
+      Object.fromEntries(
+        filteredWorlds
+          .filter(({ id }) => (venueId === id ? false : true))
```

Comment:
same comment about removing the ternary and the negation afterwards as above

Reply:
done 97a5880